language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/TransactionGenerator.scala

```diff
@@ -240,7 +240,6 @@ object TransactionGenerator {
       createdEventBlob,
       interfaceViews.view.collect { case (_, (id, Right(rec))) => (id, rec) }.toMap.asJava,
       interfaceViews.view.collect { case (_, (id, Left(stat))) => (id, stat) }.toMap.asJava,
-      None.toJava, // agreementText, to be removed
```

Comment:
Why can't this be removed straight away?

Reply:
We will remove from the proto in a second step. 
I will take care of that.

Reply:
This one is actually removed. There are remainings in other places, though.